tsdoc/src/configuration/TSDocConfiguration.ts

```diff
@@ -182,6 +192,33 @@ export class TSDocConfiguration {
     }
   }
 
+  /**
+   * Overwrite the supported HTML elements.
+   */
+  public setSupportedHtmlElements(htmlTags: string[]): void {
+    this._supportedHtmlElements.clear();
+    for (const htmlTag of htmlTags) {
+      this._supportedHtmlElements.add(htmlTag);
+    }
+  }
+
+  /**
+   * Overwrite the validator's existing value for `reportUnsupportedHtmlElements`.
+   */
+  public setReportUnsupportedHtmlElements(reportUnsupportedHtmlElements: boolean): void {
+    this.validation.reportUnsupportedHtmlElements = reportUnsupportedHtmlElements;
+  }
+
+  /**
+   * Returns true if the html element is supported in this configuration.
+   */
+  public isHtmlElementSupported(htmlTag: string): boolean {
+    if (this._supportedHtmlElements.size === 0) {
+      return true;
+    }
```

Comment:
This seems incorrect. We said there's a valid case where HTML is forbidden entirely, which is represented by an empty set.

Reply:
fixed